### PR TITLE
Local Transport

### PR DESCRIPTION
**Pull request by joker234**
* Add information on City-Ticket
* Add venue link to public transport information

### CODE REVIEW

File: attending/index.html [hunk 1]
```diff
@@ -43,6 +35,16 @@ <h3 class='space-bottom1'>Public Transport Map</h3>
...
+        <p>If you want to buy public transport tickets for the days before or after the conference, you might use the vending machines found at most stations. When using busses you can also buy tickets directly from the bus driver. When travelling within Heidelberg, you most likely need to buy a ticket at "Preisstufe 2" (also called “Großwabe Heidelberg”), which costs € 2.60 for a single ticket or € 7.00 for a day ticket. Prices for the day fare get cheaper if you travel in groups of up to 6 people. If you want to buy your ticket with your smartphone, you can use the <a href="https://play.google.com/store/apps/details?id=de.hafas.android.db">Deutsche Bahn app “Navigator”</a> that sells not only train tickets, but also local transport tickets at many places in Germany.</p>
+
+        <p>If you are travelling from / to Heidelberg by train, check out if you have a +City on your DB train ticket. That means that your train ticket includes a <a href="https://www.bahn.de/en/view/offers/germany/tickets-for-local-transport.shtml">City-Ticket</a> and is therefore also valid for a single ride in Heidelberg to your final destination.</p>
```
Comment (Nakaner) [inline attending/index.html]: Please add "The city ticket option is valid on the first day of the validity of the ticket only."

This means that it is not valid after 3am if one arrives at night.

This is not mentioned on the linked page but part of section 3.5 of the Beförderungsbedingungen Personenverkehr ([current version](https://www.bahn.de/p/view/mdb/bahnintern/agb/gesamt_2019/mdb_298620_befrderungsbedingungen_der_db_ag_-_stand_16_09_2019.pdf))

File: attending/vrn_map.js [hunk 2]
```diff
@@ -34,9 +34,9 @@ function load_map() {
         [48.94, 7.24],
         [49.8, 10.22]
     ]);
-    L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
-      attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, map tiles © <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA 2.0</a>',
-      maxZoom: 19
+    L.tileLayer('https://tileserver.memomaps.de/tilegen/{z}/{x}/{y}.png', {
```
Comment (Nakaner) [inline attending/vrn_map.js]: Please revert this change. ÖPNVKarte is not mentioned in the OSMF Privacy Policy.

Did you check that public transport mapping in OSM in Heidelberg is even useful at all? I mean the routes, not the stops.

**Review by Nakaner — CHANGES_REQUESTED** [review]

**Comment by Nakaner** [comment]
@joker234 If you are already writing advice related to public transport, please mention that VRN tickets sold by RNV vending machines (trams), have to be stamped (in the bus, tram or at a DB station). VRN Tickets from DB machines do not require stamping, they are valid from the moment of purchase.

File: attending/vrn_map.js (hunk 2 above)
Comment (joker234) [reply attending/vrn_map.js]: Yes, thanks for the hint. Didn't think about that.

File: attending/index.html (hunk 1 above)
Comment (joker234) [reply attending/index.html]: Technically right. Didn't want to overstrain the foreign readers. l Added a sentence, hope it is as you wished :)

File: attending/vrn_map.js (hunk 2 above)
Comment (joker234) [reply attending/vrn_map.js]: > Did you check that public transport mapping in OSM in Heidelberg is even useful at all? I mean the routes, not the stops.

[whb](https://www.openstreetmap.org/user/whb)  and I updated the area around the Hbf after finishing the tram & bus station. All tram and the most important bus lines should be correct again around the central station. The tiles öpnvkarte version should be fine now, routes and stations get updated until the SotM afaik: http://öpnvkarte.de/#8.6755;49.4039;17

**Review by Nakaner — APPROVED** [review]
